src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml

```diff
@@ -7,7 +7,7 @@
              mc:Ignorable="d" 
             Background="LightGray"
             BorderThickness="1"
-            BorderBrush="SlateGray"
+            BorderBrush="DarkGray"
```

Comment:
Not important now, but it would be nice to use the system accent color for the border.